setup.cfg

```diff
@@ -15,7 +15,7 @@ classifiers =
 packages = find:
 python_requires = >=3.6
 install_requires =
-    python-keycloak == 1.4.0
+    python-keycloak
```

Comment:
Having a version restriction (with specifications, similar to the other dependencies) would still be advisable

Reply:
Then I would go for >=1 and <4.

Reply:
If there where some breaking changes between Keycloak versions, we might need to limit it (for example higher than 3). So older Keycloak versions use shieldapi <2, newer >=2

Reply:
In that case, compatibility matrix between shield api and keycloak will be needed. See here an example: https://github.com/scttcper/ngx-toastr?tab=readme-ov-file#dependencies

Reply:
Since this compatibility is dictated from [python-keycloak](https://github.com/marcospereirampj/python-keycloak), we would need this input from this specific project. However, this does not exist on the [official docs](https://pypi.org/project/python-keycloak/) of the package. Therefore, we would have to investigate on our own, which might be some overhead.

Reply:
Alternatively, we can run some pytest over some different python-keycloak versions automatically and see if the routes of the keycloak-mock are compatible with the package-dependency.

Reply:
> Since this compatibility is dictated from [python-keycloak](https://github.com/marcospereirampj/python-keycloak), we would need this input from this specific project. However, this does not exist on the [official docs](https://pypi.org/project/python-keycloak/) of the package. Therefore, we would have to investigate on our own, which might be some overhead.

That's a good point. I think we can go ahead with you latest change, and deal with the failures as they come ;)

Reply:
I think we could just test if our current code (which runs on older Keycloak versions) works when shieldapi uses the latest python-keycloak. If so, they are backwards compatible, and there is no need for such a matrix

Reply:
@pablo-de-andres, would you like to test it manually and approve then ?